### DISCUSSION TITLE
[auth/unknown] - java.security.cert.CertPathValidatorException

### DISCUSSION THREAD

**Discussion by asherguedalia** [invertase/react-native-firebase#5995] [Q&A] [answered]
Does anyone know what the cause of this could be? 

 [auth/unknown] An internal error has occurred. [java.security.cert.CertPathValidatorException: Trust anchor for certification path not found].

Some of my users get it when trying to receive an sms code: auth().signInWithPhoneNumber(phoneNumber);

**Comment by mikehardy** [comment, marked as answer]
Hi there!
Unfortunately this will not be solved by anything in react-native-firebase, it is not actionable here.

If there was something *terribly* wrong with google's firebase network infrastructure, that *could* cause this, but that is not the problem. That would be like firebase going down for all android users at once - we'd all know about it.

The cause of this is some sort of firewall or proxy likely trying to do a man-in-the-middle transparent SSL redirection with an untrusted certificate or similar.

See https://github.com/firebase/firebase-android-sdk/issues?q=java.security.cert.CertPathValidatorException
